### PR TITLE
Update readme, add more examples

### PR DESCRIPTION
**Pull request by davidrusu**
[rendered](https://github.com/rust-crdt/rust-crdt/blob/update-readme/README.md)
And a new logo!!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,29 +2,30 @@
...
-- [x] LWW Register
-- [x] G-Counter
-- [ ] Top-K Set
-- [ ] Map
-- [ ] G-Set
-- [ ] OR-Set
-- [x] PN-Counter
-- [ ] EM-Counter
 
+#### Things we still need to build (PR's welcome)
```
Comment (lucab) [inline README.md]: While I understand both items, it is a bit less clear to me what are your wishes and which kind of API you envision. Would you mind opening a ticket for each and then linking them here?

Comment (davidrusu) [reply README.md]: I'll move these to issues, I'm not sure it needs to stay in the README, maybe I'll bring it back..

File: README.md [hunk 2]
```diff
@@ -2,29 +2,30 @@
...
-Thoroughly tested serializable practical CRDT's ported from riak_dt.
-
+A family of thoroughly tested CRDT's.
 
 [documentation](https://docs.rs/crdts/1.2.11/crdts/)
 
+- **VClock**: Vector clock. Typically used to track the causal history
+- **ORSWOT**: Observed-Remove Set Without Tombstones. An add-biased set
+- **Map**: Add biased Map with reset-remove semantics. Map values are also CRDT's
+- **MVReg**: Multi-Value Register. Holds a value, concurrent edits are delt with by storing both.
```
[the remark this answers is not in the document]
Comment (davidrusu) [reply README.md]: got rid of this section, I didn't like how much space it took up

**Comment by davidrusu** [comment]
@lucab want to look over it again? see the rendered readme here: [rendered](https://github.com/rust-crdt/rust-crdt/blob/update-readme/README.md)

@spacejam whadaya think of the logo?

**Comment by lucab** [comment]
Sorry, I missed the previous notification. Can you maybe commit the SVGs for the graphics too?

**Pull request merged**